apps/nouns-camp/src/contracts.js

```diff
@@ -140,6 +140,8 @@ const metaByIdentifier = {
   },
 };
 
+export const CAMP_CLIENT_ID = 3;
```

Comment:
1. I didn't register a clientId in Sepolia, but thought it was fine to use the same
2. not sure about the case convention in JS for exposed constants. can make it campClientId if better